backend/app/api/router.py

```
@@ -1,26 +1,33 @@
+    )
+
+    router.include_router(
+        integrations_router,
+        prefix="/v1/integrations",
+        tags=["Integrations"]
+    )
+    
+    return router
 
-__all__ = ["api_router"]
+__all__ = ["core_router", "auth_router"]
```

Ruff (0.14.14)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)